lib/embed.dart

```diff
@@ -733,6 +733,7 @@ class Embed {
 
   void _resetCode() {
     setContextSources(lastInjectedSourceCode);
+    Timer.run(_performAnalysis);
```

Comment:
Does this still work without `Timer.run`, or is this to account for a timing issue?

Reply:
It likely works without it, I just tested this to match what we do in the normal playground and kept it in case. The comment there has some background on its necessity.

I won't have time to test without it today so feel free to test without it and commit to this branch if desired. Otherwise I can get to it tomorrow.

Reply:
No problem, thanks for fixing this.